loops_lab.js

```diff
@@ -0,0 +1,137 @@
+/*
+1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
```

Comment:
I would recommend solving bonus questions next time. It will help you with practice and also it's generally good learning practice. I think you understood what loops are and how to implement them. Also try to practice more of for loops as you will use them often.